### PR TITLE
Added vectors for okta, uptimerobot, pushover and matomo

### PR DESCRIPTION
**Pull request by svenkubiak**
New vectors for okta, uptimerobot, pushover and matomo.

No fixed width and height and I ran them through the JSFiddle. Hope that worked.

### CODE REVIEW

**Comment by svenkubiak** [comment]
Thanks for fixing the viewbox. Regarding the extra input field. Which SVG or information.json is missing?

**Review by tijme — CHANGES_REQUESTED** [review]

File: vectors/matomo/matomo.svg [hunk 1]
```diff
@@ -0,0 +1,236 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" enable-background="new 0 0 360 360" xml:space="preserve" viewBox="Infinity Infinity -Infinity -Infinity">
+<image overflow="visible" xlink:href="data:image/jpeg;base64,/9j/4QAYRXhpZgAASUkqAAgAAAAAAAAAAAAAAP/sABFEdWNreQABAAQAAABQAAD/4QMraHR0cDov
```
[the remark this answers is not in the document]
Comment (svenkubiak) [reply vectors/matomo/matomo.svg]: I remove the vector as there seems to be no version without an embeeded image.

File: vectors/okta/okta.svg [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" style="enable-background:new 0 0 400 134.7;" xml:space="preserve" viewBox="Infinity Infinity -Infinity -Infinity">
```
[the remark this answers is not in the document]
Comment (svenkubiak) [reply vectors/okta/okta.svg]: Replaced the infitiy with the original values. Hope that works.

File: vectors/pushover/pushover.svg [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="-61.77 -61.80 838.78 838.78">
```
[the remark this answers is not in the document]
Comment (svenkubiak) [reply vectors/pushover/pushover.svg]: Took the image from the project homepage (https://support.pushover.net/i63-pushover-logos-and-usage). If that doesnt work, we need to remove it as well - couldnt find another version.

**Pull request merged**